### DISCUSSION TITLE
Points on the edge get cut off when zooming

### DISCUSSION THREAD

**Discussion by rbb81** [chartjs/chartjs-plugin-zoom#643] [Q&A] [answered]
Hi, 

## Expected behavior
All points should be properly rendered on the map when zooming
![image](https://user-images.githubusercontent.com/61752013/158783253-50967200-cfaa-416c-bc7d-ea3ab32a55b8.png)

## Current behavior
Points on the edges of the graph get cut off when zooming
![image](https://user-images.githubusercontent.com/61752013/158783357-33a6197e-89e6-43ca-ba62-a7fdf1afb5dc.png)

## Steps to Reproduce
You can go to the official [documentation](https://www.chartjs.org/chartjs-plugin-zoom/latest/guide/usage.html) site and try the zoom functionality.

## Environment
Chart.js version: ^3.5.1
Chartjs-plugin-zoom version: ^1.2.0

**Comment by kurkle** [comment, marked as answer]
This is chart.js `clip` behavior, see https://www.chartjs.org/docs/latest/charts/line.html#general

**Reply by rbb81** [reply]
Thank you 💯
